platform-configs/msi-pro-z690-a-wifi-ddr4.robot

```diff
@@ -17,3 +17,15 @@ ${WIRELESS_CARD_BLUETOOTH_SUPPORT}=     ${TRUE}
 
 ${CPU_MAX_FREQUENCY}=                   5000
 ${CPU_MIN_FREQUENCY}=                   300
+
+${DEVICE_AUDIO2}=                       Raptorlake HDMI
+
+${DEF_THREADS_PER_CORE}=                2
+${DEF_THREADS_TOTAL}=                   28
+${DEF_ONLINE_CPU}=                      0-27
+${DEF_SOCKETS}=                         1
+
+${DEF_CORES_PER_SOCKET}=                20
+
+${CPU_P_CORES_MAX}=                     8
+${CPU_E_CORES_MAX}=                     12
```

Comment:
We have two such boards in the lab, but these are valid for only one of them. For now, please add a comment mentioning this fact and CPU model for which those values are valid, we will have to somehow handle it in the future. The other platform isn't fully remotely available anyway.

Reply:
@JanPrusinowski broken link ^

Reply:
Works for me... But I edited it so that it is wraped.